From 917ca5a6ebd22d94b59bf9b2eb11b9a4f1e9885b Mon Sep 17 00:00:00 2001
From: Mike Purvis <mpurvis@clearpathrobotics.com>
Date: Thu, 5 Mar 2015 10:45:44 -0500
Subject: [PATCH] Patched in installation rule for package.xml.

---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 82a117b..05af220 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -629,3 +629,7 @@ ENDIF( ACADO_WITH_EXAMPLES )
 IF( ACADO_INTERNAL )
 	ADD_SUBDIRECTORY( ./experimental )
 ENDIF( ACADO_INTERNAL )
+
+
+# Patched in installation rule.
+install(FILES package.xml DESTINATION share/acado)
-- 
2.0.4

